### ISSUE TITLE
AVIDEO accessibility plugin needed.

### ISSUE THREAD

**Issue by guymass** [WWBN/AVideo#6578]
The problem is we need an accessibility plugin or a way to handle the site for people who require this. To my understanding, a site which does not have this support can incur a law suite up to few $15-$20K. We would like to sort this issue to prevent any such legal problems in the future. 

For example, Wordpress has a plugin that does this easily. Also we have for example this site which can install such a module on our site, however I do not want to give access to anyone else into my server and prefer this solution comes from you guys.

The Wordpress plugin will set an icon on the right side for example to allow people to change the site to fit their needs. People will need this and I think AVideo should take this into consideration in order to provide protection for its platform users.

![Screen Shot 5782-09-10 at 19 20 47](https://user-images.githubusercontent.com/58882903/167899046-4364a36f-db65-42d6-bd10-c650e641452f.jpg)

Please let me know what you think and how we can resolve this matter.

Much thanks for all the great support

**Comment by guymass** [comment]
It will open a side menu which allows the user to change how the page looks and feels to fit their needs like, marking headings, changing the background color, zooming in and out, enlarging fonts etc...

I only have this now in Hebrew menu didn't translate it yet, but I am sure WP and other sites have such things in english. This is basically an easy way for a user to change how the site looks and is important to allow for people who are impaired in any way.

This is a screen shot of my menu, sorry its in Hebrew but I will translate here what is listed on it:
![Screen Shot 5782-09-12 at 13 42 12](https://user-images.githubusercontent.com/58882903/168268332-f56bfbe7-1d4c-4787-a793-107c0c315258.jpg)

1. Stop flashing ads or other flashes on page
2. Mark Headlines
3. Change background color
4. Zoom In
5. Zoom Out
6. Minimize Font
7. Maximize Font
8. Set a readable Font
9. High Contrast
10. Low Contrast
11. Add underlines to links/URLs
12. Mark Links/Urls

**Comment by DanielnetoDotCom** [comment]
I found this 

https://github.com/paypal/bootstrap-accessibility-plugin/

that may work for us, but honestly, I did not see any difference on their demo page

https://paypal.github.io/bootstrap-accessibility-plugin/demo.html

am I missing something?

**Comment by guymass** [comment]
This is not the Accessibility you need, it is not a PayPal thing, this is for the whole site in general.

**Comment by guymass** [comment]
Maybe even convert a WP plugin to something compatible with AVideo.
https://wpaccessibility.io/

**Comment by DanielnetoDotCom** [comment]
Hi, the link I sent is not for Paypal only, this may work for all sites (but I didn't see any difference)

but the link https://wpaccessibility.io/ seems to be possible to do.

I may add this in future releases.

**Comment by guymass** [comment]
Great,  god bless, we will be waiting.
